daemon/command/config_unix.go

```diff
@@ -51,6 +51,7 @@ func installConfigFlags(conf *config.Config, flags *pflag.FlagSet) {
 	flags.Var(&conf.ShmSize, "default-shm-size", "Default shm size for containers")
 	flags.BoolVar(&conf.NoNewPrivileges, "no-new-privileges", false, "Set no-new-privileges by default for new containers")
 	flags.StringVar(&conf.IpcMode, "default-ipc-mode", conf.IpcMode, `Default mode for containers ipc ("shareable" | "private")`)
+	flags.IntVar(&conf.NetworkConfig.DefaultSubnetSize, "default-subnet-size", 24, "Default size for IPv4 subnets. This option is a no-op if the feature 'global-default-subnet-size' is not enabled")
```

Comment:
Not sure about saying "no-op" in the help text, maybe ...

```suggestion
	flags.IntVar(&conf.NetworkConfig.DefaultSubnetSize, "default-subnet-size", 24, "Default size for IPv4 subnets, if feature 'global-default-subnet-size' is enabled")
```